app/assets/javascripts/discourse/routes/topic-from-params.js.es6

```diff
@@ -36,6 +36,11 @@ export default Discourse.Route.extend({
         // TODO we are seeing errors where closest post is null and this is exploding
         // we need better handling and logging for this condition.
 
+        // there are no closestPost for hidden topics
+        if (topic.topic_hidden) {
```

Comment:
`topic.topic_hidden` is a little repetitive. Could we name it `topic.hidden` or perhaps `topic.view_hidden`?